<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget{
	background-color: #222831;
}
#sidebarFrame{
	
	background-color: #31363F;
}

QPushButton{
	background-color: #316669;
	border: none;
 	border-radius: 6px;
  	color: #EEEEEE;
  	font-size: 14px;
  	font-weight: 500;
  	line-height: 20px;
  	list-style: none;
  	padding: 4px 12px;
  	height:30px;
}
QPushButton:hover{
	background-color: rgba(118, 171, 174, 203);
 	text-decoration: none;
}
QPushButton:focus{
 	outline: 1px transparent;
}
QLabel{
	color: #EEEEEE;
	font-weight: bold;
	font-size: 16px;
}
QGroupBox{
	color: #EEEEEE;
	font-weight: bold;
	font-size: 16px;
	padding:30px 0px 0px 0px;
	border:none;	
}
QLineEdit{
	background-color: #31363F;
	border:none;
	padding:4px 10px;
	border-radius: 3px;
}
QFrame{
	border:none;
}
</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="sidebarFrame">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	
  	height:40px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="logo">
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>203</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="segmentationBtn">
         <property name="text">
          <string>Segmentation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="filteringBtn">
         <property name="text">
          <string>Filtering</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="edgeDetectionBtn">
         <property name="text">
          <string>Edge Deteiction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="equalizationBtn">
         <property name="text">
          <string>Equalization</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="histogramsBtn">
         <property name="text">
          <string>Histogram Plots</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="thresholdingBtn">
         <property name="text">
          <string>Thresholding</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="hybridBtn">
         <property name="text">
          <string>Hybrid Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="frequencyBtn">
         <property name="text">
          <string>Frequency filter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="houghTransformBtn">
         <property name="text">
          <string>Hough Transform</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ActiveContourBtn">
         <property name="text">
          <string>Active Contour</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="matchingBtn">
         <property name="text">
          <string>Matching</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="harrisBtn">
         <property name="text">
          <string>Harris Operator</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="siftBtn">
         <property name="text">
          <string>SIFT</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>202</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="widgetsLayout">
      <item>
       <widget class="QStackedWidget" name="stackedWidget"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
